### ISSUE TITLE
Make KJ support emscripten fibers

### ISSUE THREAD

**Issue by unicomp21** [capnproto/capnproto#1049]
https://emscripten.org/docs/api_reference/fiber.h.html?highlight=emscripten_fiber_swap

The intention being to get all this going in a Dockerfile

**Comment by kentonv** [comment]
Hmm I think first we have to support emscripten at all, right?

**Comment by unicomp21** [comment]
I think it already works!

**Comment by kentonv** [comment]
Cool, then we should add it to CI to make sure it keeps working. :)

**Comment by unicomp21** [comment]
How far do we want to go?  Build something and run compiled emscripten/webassembly tests in headless chrome?

**Comment by kentonv** [comment]
Yeah, I'd want to see all the tests run in some form. Headless Chrome is one option. I think it might also be possible to run them under Node.js? Whatever is easiest.
